### ISSUE TITLE
Unconscious Issue

### ISSUE THREAD

**Issue by R4Z0R49** [DayZMod/DayZ#630]
Currently there is a bug with unconscious and making the player invisible on recover.

**Comment by Detoeni** [comment]
saw something similar to this on the fragnet server with the new version, but in our case the player was visible but at the location where they fell unconscious\* while they were in fact running around somewhere else.

*visible player was prone, conscious and could be interacted with as normal.

**Comment by Lukethero** [comment]
Yeah huge issue, this needs to be fixed ASAP. I did nto see anything about this in the changelog, but it appears to have like a couple new knockout animations, one where your arms are out like a bird, and another in like a fetal position. Whatever changed this might be the source of this.

Also on Loggout it looks like your guy dies and sinks into the ground. OBviosly you do not actually die, jsut something I noticed. Not sure what that is about, but seems like it could be related.
